src/acl/Checklist.cc

```diff
@@ -155,17 +154,20 @@ ACLChecklist::goAsync(AsyncStarter starter, const Acl::Node &acl)
 // ACLFilledChecklist overwrites this to unclock something before we
 // "delete this"
 void
-ACLChecklist::checkCallback(const Acl::Answer &answer)
+ACLChecklist::checkCallback(const char * const abortReason)
 {
+    if (abortReason)
+        markFinished(ACCESS_DUNNO, abortReason);
+    Assure(finished());
+
     ACLCB *callback_;
     void *cbdata_;
-    debugs(28, 3, "ACLChecklist::checkCallback: " << this << " answer=" << answer);
```

Comment:
The information reported by the removed debugs() is already logged by markFinished(), which should now be always called. Thus, this PR is not dropping this useful info.  The debus() call would have to be edited anyway...

P.S. This comment annotates the changes without requesting any.